sdk/test/utxo.test.ts

```diff
@@ -14,7 +14,9 @@ describe('UTXO Tests', () => {
             // P2WPKH: https://blockstream.info/address/bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq
             'bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq',
             // P2SH-P2WPKH: https://blockstream.info/address/3DFVKuT9Ft4rWpysAZ1bHpg55EBy1HVPcr
-            '3DFVKuT9Ft4rWpysAZ1bHpg55EBy1HVPcr',
+            // TODO: Use a real P2SH-P2WPKH address
+            // TODO: Add the pubkey to allow spending from the outputs
+            // '3DFVKuT9Ft4rWpysAZ1bHpg55EBy1HVPcr',
```

Comment:
Should these be addressed before merging?

Reply:
yeah, it is a test case that Dom disabled.